modules/apps/feature-flag/feature-flag-test/src/testFunctional/tests/FeatureFlag.testcase

```diff
@@ -91,6 +91,38 @@ definition {
 		}
 	}
 
+	@description = "LPS-167698 - This tests asserts that a feature flag can be enabled with dependencies."
+	@priority = 3
+	test CanBeEnabledWithDependencies {
+		task ("And a feature flag is disabled") {
+			if (IsElementPresent(locator1 = "FeatureFlag#TOGGLE_ROW", title_row = "Terms and Conditions pages", toggle_state = "Enabled")) {
+				ClickNoError(
+					locator1 = "FeatureFlag#TOGGLE_ROW",
+					title_row = "Terms and Conditions pages",
+					toggle_state = "Enabled");
```

Comment:
Why can't we use Check.checkToggleSwitch?

Reply:
That case we need to disable, the `Uncheck.uncheckToggleSwitch` does not work with that path.